### ISSUE TITLE
When closing the incident send the timeline (state) to those involved

### ISSUE THREAD

**Issue by danielleqa** [ResultadosDigitais/hellper#21]
### Feature Description

When closing the incident channel through the command ```@bot-name close```, the channel participants should receive the incident timeline.

**Comment by tiagotor** [comment]
> Maybe we should send the same return form `/hellper_status` as a private message to the incident commander

I agree with this. Then, we should put the return of hellper_status in the Post Mortem Document. What do you think about it? We can replace the it in the document after each /hellper_status in the channel? Just an idea.

**Comment by GLOFonseca** [comment]
I agree, but I think we can have a separate issue for it, they are different problems.

**Comment by fvbaltar** [comment]
I could not understand the problem, or would it be only an improvement?
Maybe, we could only display the status automatically in the channel when the incident is closed or resolved, maybe pinning the commander. 
However, I still not see the problem hehehe.

Indeed, the idea of sending the status automatically to the PostMortem Doc looks like a great idea. Also, I think it is another problem and it shall be discussed in another issue though.
